src/Npgsql/ConnectorPool.cs

```diff
@@ -49,30 +49,27 @@ sealed class ConnectorPool : IDisposable
         readonly int _min;
 
         readonly NpgsqlConnector?[] _idle;
```

Comment:
There are no reads from this array, so it can be safely removed.

Reply:
_idle is being read by TryFastAllocte

Reply:
I see now.